package.json

```diff
@@ -91,6 +95,11 @@
     "pretest": "npm run build",
     "test": "npm run lint && karma start",
     "release": "./scripts/release.sh",
-    "start": "grunt"
+    "serve": "./scripts/open.js && parallelshell 'npm run watch' 'http-server -p 5678'",
```

Comment:
the watcher and the server both need their own shell because they block subsequent commands from finishing so using `&&` doesn't work but `&` does. Here is a handy reference:

* `thing-1 && thing-2` - run `thing-1` then run `thing-2`
* `thing-1 & thing-2` - run `thing-1` and `thing-2` at the same time and combine the output